### PR TITLE
Enhancing the SDK repairVAA functionality for multipleChain

### PR DESCRIPTION
**Pull request by wasif28**
Enhancing the SDK repairVAA functionality for multipleChain out of the box to support all chains for MAINNET, TESTNET, DEVNET

### CODE REVIEW

File: sdk/js/src/utils/repairVaa.ts [hunk 1]
```diff
@@ -17,14 +15,19 @@ interface GuardianSetData {
 }
 
 export async function getCurrentGuardianSet(
-  provider: ethers.providers.JsonRpcProvider
+  provider: ethers.providers.JsonRpcProvider,
+  chainId: ChainId,
```
Comment (kev1n-peters) [inline sdk/js/src/utils/repairVaa.ts:19]: I think we should add support for non-EVM chains or at least set the type here to `EVMChainId`.

**Comment by wasif28** [comment]
@kev1n-peters  have done the suggested changes kindly give it a glance

**Review by evan-gray — CHANGES_REQUESTED** [review]
@wasif28 can you provide some background as to why this function needs to provide support for other chains or environments? Wormhole currently uses Ethereum as it's source of truth for guardian sets, so my understanding was this function was meant to match that (as it's what the guardian uses).

Furthermore, there is only one guardian set in devnet and testnet, so what is the rationale for adding those to this function?

**Comment by evan-gray** [comment]
Closing per my comment above. Feel free to open a discussion regarding your use case.

**Pull request closed without merging**